client/app/deskaas/deskaas-details/deskaas-details.html

```diff
@@ -1,165 +1,228 @@
 <div ui-view class="cui-page">
-    <cui-page-header data-title-text="DeskaasDetailsCtrl.details.alias !== 'noAlias' ? DeskaasDetailsCtrl.details.alias : DeskaasDetailsCtrl.details.serviceName"
-                 data-subtitle-text="DeskaasDetailsCtrl.details.alias !== 'noAlias' ? DeskaasDetailsCtrl.details.serviceName :''">
+    <cui-page-header data-title-text="$ctrl.details.alias !== 'noAlias' ? $ctrl.details.alias : $ctrl.details.serviceName"
```

Comment:
You could replace it with a `oui-page-header`

Reply:
Yes, it will be replaced but not as part of this PR/task. We are taking a component approach to replace all cui-components with oui-components. As part of this task, we are only replacing all cui-tile and its child components with respective oui components.